pyang/plugins/uml.py

```diff
@@ -62,6 +62,11 @@ def add_opts(self, optparser):
                                  dest="uml_inline",
                                  default =False,
                                  help="Inline groupings where they are used."),
+            optparse.make_option("--uml-no-inline-groupings-from",
```

Comment:
Although suggested by myself, I am not totally convinced by this name. 
Maybe others have better suggestions.